packages/ibm-products/src/components/TagSet/TagSetOverflow.tsx

```diff
@@ -133,6 +138,39 @@ export const TagSetOverflow = React.forwardRef(
       }
     };
 
+    const handleOverflowClick = () => {
+      // If a custom overflow function is provided then trigger that function
+      // on clicking the overflow
+      if (onOverflowClick) {
+        onOverflowClick(overflowTags);
+      } else {
+        setPopoverOpen?.(!popoverOpen);
```

Comment:
i think this was an oversight on our end, but can you do me a favor and make `setPopoverOpen` to be a required prop? it shouldn't be optional.

Reply:
Sure!